README.md

```diff
@@ -1,14 +1,18 @@
 # depstat
 
-`depstat` is a CLI for analyzing dependencies of Go modules enabled projects. 
+`depstat` is a command-line tool for analyzing dependencies of Go modules enabled projects. 
 
 ![depstat demo with k8s repo](./depstat-k8s-demo.gif)
 
 ## Installation 
-Currently, to test `depstat` out you can generate the binary directly by running `go build` in the project directory and then pointing to the location of the binary from the project whose dependencies you want to analyze. Releases will be supported once the repository for `depstat` gets finalized.
+To install depstat you can run
+
+```
+go get github.com/kubernetes-sigs/depstat@latest
+```
 
 ## Usage
-`depstat` can be used as a standalone command-line application. It is also to be used in the [Kubernetes](https://github.com/kubernetes/kubernetes) CI pipeline. For more info on the latter see [#100792](https://github.com/kubernetes/kubernetes/pull/100792).
+`depstat` can be used as a standalone command-line application. You can navigate to your go modules enabled project and use `depstat` to produce metrics about your project. Another common way to run `depstat` is in the CI pipeline of your project. This would help you analyze the dependency changes which come with PRs. You can look at how this is done for the [kubernetes/kuberenets](https://github.com/kubernetes/kubernetes) repo using [prow](https://github.com/kubernetes/test-infra/tree/master/prow) [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml). 
```

Comment:
nit: would be good to break this down into multiple lines foe easier review.

> You can navigate to your go modules enabled project and use `depstat` to produce metrics about your project.

nit: maybe rephrase as `You can use depstat to produce metrics about the dependencies of your go modules enabled project.`?

> You can look at how this is done for the [kubernetes/kuberenets](https://github.com/kubernetes/kubernetes)

nit: typo in kuberenets :)